README.md

```diff
@@ -8,3 +8,15 @@ This is not its documentation.
 - the **source code** is hosted at http://llvm.org/, and mirrored at
   https://github.com/llvm/llvm-project/tree/master/clang-tools-extra/clangd.
 - the **website source code** is at https://github.com/clangd/clangd.github.io/
+
+### Communication channels
+
+If you have any questions or feedback, you can use the following ways of
+reaching the Clangd community and developers:
+
+- #clangd room is hosted on [LLVM's Discord
```

Comment:
drop `is` and prepend with `quick chat at` ?

I would also add a link to `#clangd`, created https://discord.gg/RfF7rx4 for you